### DISCUSSION TITLE
Documentation TS 100 default firmware link update

### DISCUSSION THREAD

**Discussion by martir123** [Ralim/IronOS#1840] [Ideas]
Hello, this is proposal to update firmware download link - it redirects user to non existing page .... 
so here is actual one after a lot of research .... 
https://e-design.com.cn/en/NewsDetail/4203645.html

Even beter would be to have default .hex for various solders as restore method in case ironOS will just refuse to install and old os is gone.
EG: https://forum.minidso.com/forum.php?mod=attachment&aid=OTYyMnw1MWRmZTRlZXwxNjk2ODIzMTU5fDU2MXw4Njg%3D

and so on. no one can be sure how long those links stay valid and i guess default OSwill not change that often.

**Comment by ia** [comment]
Hello. Thanks for the link to the original firmware. We work on that.

**Comment by ia** [comment]
Official firmware has been uploaded to the supplementary repo and related documentation has been updated. So since this discussion/idea looked more like an issue report, it can be closed now, I guess.

**Reply by martir123** [reply]
sure. and thank you that would save me a few hours of time.
